Razor/HotKeys/Misc.cs

```diff
@@ -245,6 +246,19 @@ private static void AllCorpses()
             }
         }
 
+        private static void OpenAllCorpses()
+        {
+            foreach (Item i in World.Items.Values)
+            {
+                if (i.IsCorpse &&
+                    Utility.InRange(i.Position, World.Player.Position, Config.GetInt("CorpseRange")) &&
+                    World.Player != null && World.Player.Visible)
```

Comment:
Please check for null first, or it would fail on `World.Player.Position` if World is null.